crates/wash-cli/src/up/config.rs

```diff
@@ -92,27 +91,28 @@ pub async fn configure_host_env(
 
     // NATS RPC connection configuration
     if let Some(host) = wasmcloud_opts.rpc_host {
-        host_config.insert(WASMCLOUD_RPC_HOST.to_string(), host);
-    } else {
-        host_config.insert(WASMCLOUD_RPC_HOST.to_string(), nats_opts.nats_host.clone());
+        host_config.insert(WASMCLOUD_RPC_HOST.to_string(), host.clone());
+        // TODO: remove me after the host removes support for PROV_RPC connections
```

Comment:
This note and the other similar ones are probably worth adding to https://github.com/wasmCloud/wasmCloud/pull/774 so we don't forget?

Reply:
Merging #774 occupies a permanently reserved portion of my mind 😉